package.json

```diff
@@ -162,10 +167,12 @@
     "@rollup/plugin-replace": "^5.0.2",
     "@rollup/plugin-typescript": "^11.0.0",
     "@types/babel__core": "^7.20.0",
+    "@types/throttle-debounce": "~5.0.0",
     "@types/semver": "^7.3.13",
     "@types/vscode": "~1.69.1",
     "@typescript-eslint/eslint-plugin": "^5.54.0",
     "@typescript-eslint/parser": "^5.54.0",
+    "editorconfig": "~1.0.2",
```

Comment:
what is it for?

Reply:
https://editorconfig.org/

It configures tabs, line endings, see new `.editorconfig` file in root. We have that in monaco-languageclient, too.

Reply:
Do we need the vscode extension? I don't understand how installing a npm dependency can do anything related to the editor

Reply:
Yes, true, I forgot about that. Do want a `.github/extensions.json` or shall I revert this?

Reply:
What does it do? I don't see this file on monaco-languageclient

Reply:
Discussion is outdated.